cmd/plugins/memtierd/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG GO_VERSION=1.21
+
+FROM golang:${GO_VERSION}-bullseye as builder
+
+WORKDIR /go/builder
+
+RUN GOBIN=/bin go install github.com/intel/memtierd/cmd/memtierd@latest
```

Comment:
Hmm, this looks like a potential issue to produce surprises by irreproducible build results 🧐 Could we pin memtierd to some known version?

Reply:
Fixed.